guidelines/sc/22/accessible-authentication.html

```diff
@@ -6,7 +6,9 @@ <h4>Accessible Authentication</h4>
 	<p class="change">New</p>
     
     <p>For each step in an authentication process that relies on a <a>cognitive function test</a>, at least one other authentication method is available that does not rely on a cognitive function test, or a mechanism is available to assist the user in completing the cognitive function test.</p>
-    <p>Exception: When the cognitive function test is to recognize objects, or content the user provided to the website.</p>
+    
+    <p>Exception: The cognitive function tests ask the user to recognize content that the user provided to the website or to recognize objects.</p>
```

Comment:
i'd add a comma after `website`. and perhaps add some kind of qualifier about what kind of objects? "common" objects? or is that too loaded/avoided on purpose?

Reply:
I don't think you need a comma with an 'or'.
And yes, we're avoiding common objects as then it then needs defining what is common. After a long discussion, everyone agreed that we should just go with 'objects'.

Reply:
reads a bit odd without the comma, but maybe it's just me. possibly due to the second clause being very short and seemingly just dangling at the end of that sentence